test-suite/testkit/src/lib.rs

```diff
@@ -690,6 +690,8 @@ impl TestKit {
                 ApiRuntimeConfig::new(private_api_address, ApiAccess::Private),
             ],
             api_aggregator,
+            service_retry_attempt: 20,
```

Comment:
Hm... Very minor nit: not sure if we should have that many attempts in the testkit. I'd suggest 5 or even 3.

@aleksuss @alekseysidorov what's your opinion on this?

Reply:
IMO it's not principle. 5 is ok.